src/explore-education-statistics-common/src/modules/table-tool/components/FilterHierarchy.module.scss

```diff
@@ -0,0 +1,23 @@
+@import '~govuk-frontend/dist/govuk/base';
+
+.tier {
+  background: govuk-colour('light-grey');
+  padding: 0.5rem;
```

Comment:
Could you use the `govuk-spacing` for the spacings in this file and in `FilterHierarchyOptions.module.scss`?